simple/tests/stats/test_data/stat_var_hierarchy_generator/expected/basic_svgs.json

```diff
@@ -0,0 +1,92 @@
+[
+ {
+  "svg_id": "c/g/Person_Gender-Female_Race-Asian",
+  "svg_name": "Person With Gender = Female, Race = Asian",
+  "sv_ids": [
+   "sv1"
+  ],
+  "parent_svg_ids": [
+   "c/g/Person_Gender_Race-Asian",
+   "c/g/Person_Gender-Female_Race"
+  ],
+  "child_svg_ids": []
+ },
+ {
+  "svg_id": "c/g/Person_Gender_Race-Asian",
+  "svg_name": "Person With Gender, Race = Asian",
+  "sv_ids": [],
+  "parent_svg_ids": [
+   "c/g/Person_Race-Asian"
+  ],
+  "child_svg_ids": [
+   "c/g/Person_Gender-Female_Race-Asian"
+  ]
+ },
+ {
+  "svg_id": "c/g/Person_Race-Asian",
+  "svg_name": "Person With Race = Asian",
+  "sv_ids": [],
+  "parent_svg_ids": [
+   "c/g/Person_Race"
+  ],
+  "child_svg_ids": [
+   "c/g/Person_Gender_Race-Asian"
+  ]
+ },
+ {
+  "svg_id": "c/g/Person_Race",
+  "svg_name": "Person With Race",
+  "sv_ids": [],
+  "parent_svg_ids": [
+   "c/g/Person"
+  ],
+  "child_svg_ids": [
+   "c/g/Person_Race-Asian"
+  ]
+ },
+ {
+  "svg_id": "c/g/Person",
+  "svg_name": "Person",
+  "sv_ids": [],
+  "parent_svg_ids": [
+   "dc/g/Root"
```

Comment:
Should this be DC root or custom DC root (which I thought was different?)

Reply:
A [custom DC root](https://github.com/datacommonsorg/import/blob/57056eac9987dd1cb9caf0bfb176b1534a5e1cce/simple/stats/nodes.py#L46) is created if users have not specified a group for a variable. 

Should we create a custom root for the auto generated hierarchy? Can follow up with what we decide.

Reply:
Yes, to avoid confusion, it might make sense to have a separate root, and have `c/g/Root` be a child of `dc/g/Root`?

Reply:
SG. Any preference on a user friendly name of the root?